ReadIt/ReadingList/ReadingCell.swift

```diff
@@ -7,17 +7,139 @@ import UIKit
+		banner.bottomAnchor.constraint(lessThanOrEqualTo: contentView.bottomAnchor, constant: -8).isActive = true
+		banner.trailingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.trailingAnchor).isActive = true
+		banner.widthAnchor.constraint(equalToConstant: 80).isActive = true
+
+		let bannerHeight = banner.heightAnchor.constraint(equalToConstant: 80)
+		bannerHeight.priority = .defaultHigh
+		bannerHeight.isActive = true
+		
+		titleLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16).isActive = true
+		titleLabel.leadingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leadingAnchor).isActive = true
+		titleLabel.trailingAnchor.constraint(equalTo: banner.leadingAnchor, constant: -8).isActive = true
+
+		authorLabelConstraints = [
+			authorLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 6),
+			authorLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
+			authorLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor),
+			descriptionLabel.topAnchor.constraint(equalTo: authorLabel.bottomAnchor, constant: 8)
+		]
+		
+		let authorLabelSkippingConstraint = descriptionLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 8)
+		authorLabelSkippingConstraint.priority = UILayoutPriority(201)
+		authorLabelSkippingConstraint.isActive = true
 
-		self.detailTextLabel?.font = .systemFont(ofSize: 11)
-		self.detailTextLabel?.textColor = .gray
+		descriptionLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor).isActive = true
+		descriptionLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor).isActive = true
+		
+		estimatedTimeLabel.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: 6).isActive = true
+		estimatedTimeLabel.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor).isActive = true
+		estimatedTimeLabel.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor).isActive = true
+		estimatedTimeLabel.bottomAnchor.constraint(lessThanOrEqualTo: contentView.bottomAnchor, constant: -8)
+
+		let bottomBreakableConstraint = estimatedTimeLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8)
+		bottomBreakableConstraint.priority = UILayoutPriority(200)
+		bottomBreakableConstraint.isActive = true
 	}
 
 	required init?(coder aDecoder: NSCoder) {
 		fatalError("You should not use init?(coder:), ever!")
 	}
-    
+
+	func configure(_ reading: Reading) {
+		if let author = reading.author {
+			authorLabel.text = author
+			NSLayoutConstraint.activate(authorLabelConstraints)
+			layoutIfNeeded()
+		}
+
+		titleLabel.text = reading.title
+		descriptionLabel.text = reading.excerpt
+
+		let pathURL = reading.images?.first?.src
+		banner.imageFromURL(pathURL)
+		
+		if let timeToRead = reading.timeToRead {
+			estimatedTimeLabel.text = "\(timeToRead) min read"
+		}
+	}
+
+	override func prepareForReuse() {
```

Comment:
I've added this method, cause it is a good practice.

Reply:
Good catch ;)